gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Maven version
-version = 0.1.68-SNAPSHOT
+version = 0.1.70-SNAPSHOT
```

Comment:
Since there public API changes shouldn't it go to `0.1.70` per semantic versioning? I don't care either way let me know.